packages/cli-config/src/__tests__/__snapshots__/index-test.ts.snap

```diff
@@ -95,7 +95,7 @@ Object {
 }
 `;
 
-exports[`should not skip packages that have invalid configuration (to avoid breaking users): logged warning 1`] = `"warn Package react-native contains invalid configuration: \\"dependency.invalidProperty\\" is not allowed. Please verify it's properly linked using \\"npx react-native config\\" command and contact the package maintainers about this."`;
+exports[`should not skip packages that have invalid configuration (to avoid breaking users): logged warning 1`] = `"[33m[1mwarn[22m[39m [0mPackage react-native contains invalid configuration: \\"dependency.invalidProperty\\" is not allowed. Please verify it's properly linked using \\"npx react-native config\\" command and contact the package maintainers about this.[0m"`;
```

Comment:
it seems that the previous behaviour (setting `FORCE_COLOR` to `0`) also disabled formatting. I don't think pico supports this, so this text is now bold (but uncoloured).